lisa/tools/mkfs.py

```diff
@@ -102,3 +120,60 @@ def _install(self) -> bool:
             package = "btrfsprogs"
         posix_os.install_packages(package)
         return self._check_exists()
+
+
+class BSDMkfs(Mkfs):
+    _PARTITION_ID_REGEX = re.compile(r"^(?P<disk>/dev/(da|nvd)\d+)p(?P<index>\d+)$")
```

Comment:
Add examples above.

Reply:
@squirrelsc I have fixed this comment. Can you please check?